backend-project/small_eod/institutions/tests/test_validators.py

```diff
@@ -55,6 +59,9 @@ def test_regon_accepts_digits(self):
         form = f(data=dict(regon="9999999999"))
         self.assertTrue(form.is_valid())
 
+        form = f(data=dict(regon="abc1234567"))
```

Comment:
Widać jak miło i prosto dopisuje się kolejne przypadki. Może tak być jak jest. 

Z doświadczenia jednak podpowiem, że jakbyś przeniósł ```f``` do ```setUp``` / gettera to mógłbyś rozdzielić każdy przypadek na osobne testy, a wtedy widziałbyś w razie awarii więcej co nie działa. Teraz po pierwszym przypadku dostaniesz błąd i nie wiesz czy kolejne przypadki zostały uznane za poprawne, czy tylko jeden, a byłoby to przecież pomocne. Nazwa testów dokumentowałaby (tu jest to oczywiste) również co jest problemem.